cluster/aws/templates/configure-vm-aws.sh

```diff
@@ -64,6 +69,9 @@ find-master-pd() {
     attempt=$(($attempt+1))
     sleep 1
   done
+
+  # Mount the master PD as early as possible
+  echo "/dev/xvdb  /mnt/master-pd  ext4     noatime  0 0" >> /etc/fstab
```

Comment:
Not a nit!  I was trying to line it up nicely, but that got lost over time.  `man fstab` says "fields on each line are separated by tabs or spaces", I'm just going to switch to a single space in between each field.